docs/src/inputs/select/examples/exampleCreatableAdvancedSelect.jsx

```diff
@@ -0,0 +1,90 @@
+import { isEmpty } from 'lodash';
+import React, { useState } from 'react';
+import {
+    Icon,
+    Select,
+} from 'react-cm-ui';
+import { options } from '../constants';
+
+function ExampleCreatableAdvancedSelect() {
+    const [selectedValue, setOption] = useState({});
+
+    function onSelectOption(selectedOption) {
+        setOption(selectedOption);
+    }
+
+    const formatOptionLabelCustom = ({
+        key,
+        onFocus,
+        onSelect,
+        option,
+    }) => (
+        <div
+            className="Select-option"
+            key={key}
+            onClick={() => onSelect(option)}
+            onFocus={() => {}}
+            onKeyDown={() => {}}
```

Comment:
Let's use `noop` from Lodash.
I know this is just an example, but we want our examples to look like how we want actual code to be written! 😁

Reply:
Fixed

Reply:
These anonymous noop lambdas were replaced by Lodash `noop` but there is a new issue which I [commented on separately](https://github.com/saddlebackdev/react-cm-ui/pull/247#discussion_r532816497).

Resolving.